backend/src/main/java/ai/verta/modeldb/utils/ModelDBUtils.java

```diff
@@ -84,6 +81,14 @@ private ModelDBUtils() {}
     return prop;
   }
 
+  public static String appendOptionalTelepresencePath(String filePath) {
+    String telepresenceRoot = System.getenv("TELEPRESENCE_ROOT");
+    if (telepresenceRoot != null) {
+      filePath = telepresenceRoot + filePath;
```

Comment:
you should have to add File separator in between two file paths

Reply:
ideally yes, but this is just replacing the logic previously in readYamlProperties